qiskit/chemistry/algorithms/ground_state_solvers/ground_state_eigensolver.py

```diff
@@ -132,15 +132,24 @@ def evaluate_operators(self,
         # handle all possible formats of operators
         # i.e. if a user gives us a dict of operators, we return the results equivalently, etc.
         if isinstance(operators, list):
-            results = []
+            results = []  # type: ignore
             for op in operators:
-                results.append(self._eval_op(state, op, quantum_instance))
+                if op is None:
+                    results.append(None)
```

Comment:
Is the return type of `evaluate_operators()` `Optional[float]` in this case?

Reply:
Good catch, thanks! :)